packages/event-handler/tests/unit/rest/Router/error-handling.test.ts

```diff
@@ -567,3 +588,32 @@ describe('Class: Router - Error Handling', () => {
     });
   });
 });
+describe('Class: Router - proxyEventToWebRequest Error Handling', () => {
+  beforeEach(() => {
+    vi.resetModules();
+  });
+
+  it('re-throws non-InvalidHttpMethodError from proxyEventToWebRequest', async () => {
+    // Prepare
+    vi.doMock('../../../../src/rest/converters.js', async () => {
+      const actual = await vi.importActual<
```

Comment:
I very much dislike tests like this. The reason we have to do this is because the type guards before this code path are so strict that this code is virtually unreachable. I have to mock the `proxyEventToWebRequest` thorw an exception that it will almost certainly never throw in real use.